hibernate-core/src/main/java/org/hibernate/engine/spi/SharedSessionContractImplementor.java

```diff
@@ -381,6 +381,10 @@ default String bestGuessEntityName(Object object, EntityEntry entry) {
 
 	boolean isCriteriaCopyTreeEnabled();
 
+	boolean getNativeJdbcParametersIgnored();
```

Comment:
```suggestion
	boolean isNativeJdbcParametersIgnored();
```

Reply:
The javabeans spec doesn't force use of the 'is' prefix, but states that one *can* use 'is' instead of 'get' for boolean attributes. I prefer the latterbecause the is-option is horrible.